Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppProtectionPolicyAndroid/MSFT_IntuneAppProtectionPolicyAndroid.psm1

```diff
@@ -283,7 +285,8 @@ function Get-TargetResource
         $policy.add('Apps', $appsArray)
         $policy.add('Assignments', $assignmentsArray)
         $policy.add('ExcludedGroups', $exclusionArray)
-        $policy.add('AppGroupType', $policyInfo.AppGroupType)
```

Comment:
Is appgrouptype no longer supported? If so we should look at deprecating it and print a warning if it is used.

Reply:
AppGroupTypeis is being used but is added earlier in the code, hence the duplicate entry.  I noticed that the author has made changes so will pull this out and cover the MI assignment under a separate PR.